projects/fsastorefrontlib/src/core/change-request/facade/change-request.service.ts

```diff
@@ -66,19 +64,17 @@ export class ChangeRequestService {
     );
   }
 
-  loadChangeRequest() {
+  loadChangeRequest(requestId: string) {
```

Comment:
Can you please create a follow-up task if there isn't one already for refactoring this service? Subscribe inside subscribe inside subscribe... and all that inside service.